interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Are (most) Ruby methods accessors or mutators?
 
-  Your answer:
+  Your answer: Most ruby methods ive encountered are ruby mutators. A mutator takes in data and returns however you change it using .select, .map..
 
-  Researched answer:
+  Researched answer:Most Ruby methods are in fact mutators. Mutators are called setters in ruby. Some examples of ruby setters are .each, .map, .select, .while. 
```

Comment:
Most Ruby methods are accessors. They are responsible for retrieving and packaging up data in new ways so as to not interfere with the flow of projects. .each .map .select and .while are all examples of accessors.